Classes/Domain/Model/Index.php

```diff
@@ -152,14 +155,14 @@ public function exists()
      * @param string $path
      * @param array $arguments
      * @param string $content
-     * @param boolean $prefixIndex
+     * @param bool $prefixIndex
      * @return Response
-     * @throws ElasticSearchException
+     * @throws \Exception
      */
-    public function request($method, $path = null, array $arguments = [], $content = null, $prefixIndex = true)
+    public function request(string $method, ?string $path = null, array $arguments = [], ?string $content = null, bool $prefixIndex = true): Response
     {
         if ($this->client === null) {
-            throw new Exception('The client of the index "' . $this->name . '" is not set, hence no requests can be done.');
+            throw new \Exception('The client of the index "' . $this->name . '" is not set, hence no requests can be done.', 1566313883);
```

Comment:
Ok, so the `ElasticSearchException` that was advertised was not thrown. But it's still the same (only aliased). Now `\Exception` is something else, though!

Reply:
Thanks!